src/components/picker/index.js

```diff
@@ -302,7 +303,7 @@ class Picker extends BaseComponent {
   };
 
   render() {
-    const {useNativePicker, renderPicker, customPickerProps, testID} = this.props;
```

Comment:
is this whole change really necessary? 
till now we override whatever the user pass for `enableErrors` with false. 
you can quickly just enable the user use this prop with a much smaller change

Reply:
Done